test/cover.ps1

```diff
@@ -24,7 +24,7 @@ function CommandAliasFunction
 
 Set-Alias -Name ce -Value CommandAliasFunction -Scope script
 
-$tests = $(dotnet sln $solutionFileDir list) | Select-Object -Skip 2 | Where-Object { $_ -match "^tests//" }
+$tests = $(dotnet sln $solutionFileDir list) | Select-Object -Skip 2 | Where-Object { $_ -match "^test//" }
```

Comment:
This will break existing. This will only accept test in folder `test` and not `tests`

Reply:
You might want something like `^(test|tests)//`. Don't take my word for this!